docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+## How to contribute
+
+Pocket is an open source project, and as such we welcome any contribution from anyone on the internet.
+
+Please fork, code and submit a Pull Request for the team to review and merge. We ask that you please follow the
+guidelines below in order to submit your contributions for review:
```

Comment:
NIT: There's a line breakpoint between 5 & 6, even though line 5 wraps onto a new line.

Is this intentional or do you use a specific line length in your editor?